internal/expr/bindtypes.go

```diff
@@ -115,7 +115,43 @@ func (e *memberInputExpr) bindTypes(argInfo typeinfo.ArgInfo) (any, error) {
 	if err != nil {
 		return nil, fmt.Errorf("input expression: %s: %s", err, e.raw)
 	}
-	return &typedInputExpr{input}, nil
+	return &typedInputExpr{input: input}, nil
+}
+
+// asteriskInputExpr is an input expression occurring within an INSERT
+// statement that consists of an asterisk on the left and explicit type accessors
+// on the right. This means that SQLair generates the columns.
+// e.g. "(*) VALUES ($Type1.col1, $Type2.*)".
+type asteriskInputExpr struct {
```

Comment:
Are these better named to expressly indicate part of an insert? `asteriskInsertExpr` for example.

Reply:
Renamed.